tldr.py

```diff
@@ -304,12 +282,8 @@ def main():
 
     colorama.init(strip=options.color)
 
-    if options.download_cache:
-        download_cache()
-        return
-
     if options.update_cache:
-        update_cache(remote=options.source)
+        update_cache()
         return
```

Comment:
It feels like this return should be removed, or at least while updating the help string mention that it exits after updating local cache.

Reply:
It has to be this way for commands that take no arguments. Update cache already has a print message, that updating was done or that it failed.

Reply:
You are right on that. See suggestion below on what I think is the best course of action on this point.